scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -1123,27 +1123,29 @@ <h4>Scout case analysis report</h4> - created on:&nbsp;<strong>{{report_created_
         <tr class="table-secondary">
           <th>Variant type</th>
           <th>Estimated size</th>
+          <th>Reference size</th>
           <th>Coordinates</th>
-          <th>Cytoband</th>
           <th>Gene panels</th>
         </tr>
       </thead>
       <tbody>
-        <tr>
+        {% if not variant.str_status or variant.str_status == 'normal' %}
+          <tr>
+        {% elif variant.str_status == 'full_mutation' %}
+          <tr class="table-danger">
+        {% elif variant.str_status == 'pre_mutation' %}
+          <tr class="bg-warning">
+        {% endif %}
           <td><span class="badge badge-info">{{variant.category | upper}}</span></td>
           <td>{{ variant.alternative|replace("STR", "")|replace("<", "")|replace(">", "") }}</td>
+          <td>{{ variant.str_ref }}</td>
```

Comment:
@northwestwitch ahem, and this one? 😊

Reply:
I can fix it if you want, but I'm quite sure that we'd see the app cashing before getting to the report, check here: https://github.com/Clinical-Genomics/scout/blob/a9ed7f7f050c2c8e4eee58cfe59a997d6912ac3a/scout/server/blueprints/variants/templates/variants/str-variants.html#L96

Reply:
Ok, that is another view I guess, but as long as you followed it from its controller its all good. Several of them did get default empty strings. But not all.. 😸